### ISSUE TITLE
Problem with TVM opcodes BALANCE & RANDSEED

### ISSUE THREAD

**Issue by gr8den** [ton-blockchain/ton#210]
Opcodes BALANCE & RANDSEED have incorrect GETPARAM index

Documentation (https://test.ton.org/tvm.pdf , A.11.4. Configuration primitives.) says
BALANCE equivalent to GETPARAM 6 and RANDSEED equivalent to GETPARAM 7
but in TL scheme
```
[ magic:0x076ef1ea actions:Integer msgs_sent:Integer
unixtime:Integer block_lt:Integer trans_lt:Integer
rand_seed:Integer balance_remaining:[Integer (Maybe Cell)]
myself:MsgAddressInt global_config:(Maybe Cell)
] = SmartContractInfo;
```
they have other indexes: RANDSEED is 6, BALANCE is 7

**Comment by gr8den** [comment]
fix: https://github.com/ton-blockchain/ton/pull/211

**Comment by ton-blockchain** [comment]
You are right. This will be fixed in the next commit, along with the new PRNG instructions.

**Comment by ton-blockchain** [comment]
Should be fixed in commit d41ce55305718d207867f0dac60319ec4b6846f2. Please check.
